fesvr/tsi.cc

```diff
@@ -27,16 +27,8 @@ tsi_t::~tsi_t(void)
 void tsi_t::reset()
 {
   uint32_t one = 1;
-  addr_t ipis[NHARTS_MAX];
-  int ncores = get_ipi_addrs(ipis);
 
-  if (ncores == 0) {
-      fprintf(stderr, "ERROR: No cores found\n");
-      abort();
-  }
-
-  for (int i = 0; i < ncores; i++)
-    write_chunk(ipis[i], sizeof(uint32_t), &one);
+  write_chunk(0x2000000, sizeof(uint32_t), &one);
```

Comment:
I can make it a define. Is the location of the Clint fixed in the platform spec?